auth/v1.js

```diff
@@ -134,14 +128,14 @@ api.declare({
 }, async function(req, res) {
   let clientId = req.params.clientId;
 
-  // Load client
-  let client = await this.Client.load({clientId}, true);
+  // Load client from the resolver's cache
+  let client = this.resolver.jsonClient(clientId);
```

Comment:
OK, I'll split out this refactor (it was a little fiddly to get right) into a separate PR and leave it there so it's handy if we find we want it.